### PR TITLE
centering div

### PR DESCRIPTION
**Pull request by Glazomer**

### CODE REVIEW

File: css/solution3.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+body{
+    height: 100vh;
+    position: relative;
+}
+
+.test{
+    position: absolute;
+    left: calc(50% - 50px);
+    top: calc(50% - 50px);
```
Comment (vitkarpov) [inline css/solution3.css]: What if you don't know the sizes? `transform: translate(-50%, -50%)` would help

Comment (Glazomer) [reply css/solution3.css]: > What if you don't know the sizes? `transform: translate(-50%, -50%)` would help

Great notice!

**Comment by Glazomer** [comment]
> A follow-up question: which of them do you like more and why?

the flex-box one (that what flex box intended for)

**Comment by vitkarpov** [comment]
> > A follow-up question: which of them do you like more and why?
> 
> the flex-box one (that what flex box intended for)

Yep, me too 😃

**Pull request merged**